tests/Tokenizer/CodeHasherTest.php

```diff
@@ -26,7 +26,7 @@ final class CodeHasherTest extends TestCase
 {
     public function testCodeHasher(): void
     {
-        static::assertSame('322920910', CodeHasher::calculateCodeHash('<?php echo 1;'));
-        static::assertSame('322920910', CodeHasher::calculateCodeHash('<?php echo 1;')); // calling twice, hashes should always be the same when the input doesn't change.
+        static::assertSame('d9de49676ba2316990a5acd04c8418e8', CodeHasher::calculateCodeHash('<?php echo 1;'));
```

Comment:
a collision example is https://3v4l.org/Bc1Da 😃

Reply:
I wonder how big edge case it is, as nobody complained so far. Did you faced the collision in real codebase? or only managed to generate it?

Reply:
As shown in the PR description, with large number of the files, the collision with only 32-hash space is more than expected.